packages/client/src/components/TransactionStatusIcon.js

```diff
@@ -3,17 +3,17 @@ import React from "react";
 const STATUS_TO_BACKGROUND_COLOR = {
   confirmed: "#18AA01",
   pending: "#FF8A00",
+  rejected: "#CD0B0B",
```

Comment:
Can we use here the colors defined in App.sass instead?
We have success, warning and danger classes for it